cluster-setup/upgrade-test-cluster/performance/performance_profile.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: performance.openshift.io/v1alpha1
+apiVersion: performance.openshift.io/v1
```

Comment:
upgrade test failed with error:
`error: unable to recognize "STDIN": no matches for kind "PerformanceProfile" in version "performance.openshift.io/v1"`

My assumption: in upgrade test we are using subscription channel 4.5, so may be we need to keep `v1alpha1` here?